src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/ConnectorTypes.java

```diff
@@ -27,9 +27,13 @@
 
 public class ConnectorTypes {
     @Given("Navigate to the Home Page")
-    public void navigateToTheHomePage() {
-        SeleniumDriver.openPage(Constants.WRANGLE_HOME_URL);
-        WaitHelper.waitForPageToLoad();
+    public void navigateToTheHomePage() throws Exception {
+        try {
+            SeleniumDriver.openPage(Constants.WRANGLE_HOME_URL);
+            WaitHelper.waitForPageToLoad();
+        } catch (Exception e) {
+            System.err.println(e);
```

Comment:
With this handler, will an exception fail the test or allow it to continue running?

Reply:
The build will continue.